openwisp_monitoring/device/admin.py

```diff
@@ -215,14 +215,20 @@ class Media:
         js = (
             tuple(BaseDeviceAdmin.Media.js)
             + (
-                'monitoring/js/percircle.min.js',
+                'monitoring/js/lib/percircle.min.js',
```

Comment:
**Note:** This makes it a backward incompatible change.

Reply:
@pandafy  Should I revert this change?